grpc/src/main/java/com/linecorp/armeria/internal/common/grpc/TransportStatusListener.java

```diff
@@ -24,6 +24,9 @@
  * logic through this listener, and for clients the final response {@link Status} is also returned.
  */
 public interface TransportStatusListener {
```

Comment:
This is not relevant to the change of this PR, but could you add `@FuntionalInterface` to the interface?
Could you also update the description?